source/extensions/filters/udp/udp_proxy/hash_policy_impl.cc

```diff
@@ -48,7 +50,7 @@ HashPolicyImpl::HashPolicyImpl(
     hash_impl_ = std::make_unique<KeyHashMethod>(hash_policies[0]->key());
     break;
   case UdpProxyConfig::HashPolicy::PolicySpecifierCase::POLICY_SPECIFIER_NOT_SET:
-    PANIC_DUE_TO_CORRUPT_ENUM;
+    throw EnvoyException("policy specifier not set");
```

Comment:
Why this change?

Reply:
I was trying to increase coverage for the UDP folder as a whole, and apparently, before this change this line (the PANIC) was not covered, even though there was a test that tests this scenario. I wasn't able to understand why even though a test checks this, the coverage tool considers this line as not covered, but changing to exception instead makes the trick.

Anyways, this isn't 100% related to this PR, I agree - but since it's a super small change I was hoping to piggyback this PR instead of opening a new one.

Reply:
We're actually trying to remove exceptions from the codebase. See #29840. So adding a new exception would be a step backward from that objective. Let's revert this part of the change.

Reply:
Done